tests/unit/specs/store/session.spec.js

```diff
@@ -1,26 +1,17 @@
 import sessionModule from "src/vuex/modules/session.js"
 
 describe(`Module: Session`, () => {
-  let module, state, actions, mutations
+  let module, state, actions, mutations, node
 
   const networks = [
     { id: `awesomenet`, slug: `awesome` },
     { id: `keine-ahnungnet`, slug: `ahnungnet` },
     { id: `fabo-net`, slug: `fabo` },
   ]
 
-  let mockApollo = {
-    async query() {
-      return {
-        data: {
-          networks,
-        },
-      }
-    },
-  }
-
   beforeEach(() => {
-    module = sessionModule({ apollo: mockApollo })
+    node = {}
+    module = sessionModule({ node })
```

Comment:
node? ^^

Reply:
it was there before for some reason 👀

Reply:
I will try removing it 👌

Reply:
ping?

Reply:
ouch ok. I saw you said it looks good and then I thought, ok, not so important. Will try now

Reply:
ok, yes, it was failing, because in the end it actually requires apollo, as we need it for `checkAddressRole`